arbnode/batch_poster.go

```diff
@@ -558,7 +558,15 @@ func AccessList(opts *AccessListOpts) types.AccessList {
 func (b *BatchPoster) checkEspressoValidation(
```

Comment:
The batch poster should not send any transactions to L1 if the espresso validation is not turned on because this batch poster is supposed to be running in a TEE and only be sending transactions to L1 once the espresso mode has been enabled.

The way this code is setup I think, this batch poster would still send transactions to L1 but would just not send them to hotshot. Let me know if my understanding is incorrect here